cmd/agent/app/server.go

```diff
@@ -152,6 +152,10 @@ func (a *Agent) runAdminServer(o *options.GrpcProxyAgentOptions) error {
 	if o.EnableProfiling {
 		muxHandler.HandleFunc("/debug/pprof", util.RedirectTo("/debug/pprof/"))
 		muxHandler.HandleFunc("/debug/pprof/", pprof.Index)
+		muxHandler.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
```

Comment:
You can make an argument that all the pprof methods can expose sensitive information and that is why we require it to be explicitly enabled. That having been said cmdline both seems to potentially expose things the most (eg. AuthenticationAudience and anything we unwittingly add later) but also the least useful given that the cmdline flags are already being logged at startup. Additionally its much easier to do things like mask fields we consider to be too sensitive in the logs. Much harder to do that on pprof/cmdline. I'm ok with the rest but I would prefer we not expose cmdline.

Reply:
That sounds reasonable, removed.